egret/models/tests/transmission_test_instances/acpf_solution_files/pglib_opf_case3012wp_k_acpf_solution.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
I personally think we should avoid large problems in our automated testing, but this is okay for now.

Reply:
I agree. This problem had some important features. I can look for other smaller problems that have similar features when I work on additional tests.

Reply:
The pglib-opf 300 bus case tends to be good for this, if you hadn't tried it already.